lightcycle-processor/build.gradle

```diff
@@ -7,14 +7,9 @@ checkstyle {
 }
 
 dependencies {
-  compile project(':lightcycle-api')
-  compile deps.javapoet
+  implementation project(':lightcycle-api')
+  implementation deps.javapoet
 
-  testCompile deps.android
-  testCompile deps.support_v4
-  testCompile deps.compiletesting
-  // compile-testing fails with JavacTool ClassNotFoundException, so add tools.jar manually
-  // (this should be the correct cross-platform solution to do this)
-  // cf. http://stackoverflow.com/questions/11345193/gradle-does-not-find-tools-jar
-  testCompile files(((URLClassLoader) javax.tools.ToolProvider.getSystemToolClassLoader()).getURLs())
+  testImplementation deps.appcompat_x
```

Comment:
This is an interesting one: `appcompat` is an Android library (aar) but this project is just a regular Java library. Does it actually compile? 🤔

Reply:
it was compiling, but apparently this dependency is no longer necessary.

Also missed a part with some test failures when composing virtual classes in tests, should be fixed now.